content/pages/index.html

```diff
@@ -2,6 +2,7 @@
     <head>
         <title>the leading open source test case management system</title>
         <meta name="save_as" content="index.html">
+        <meta name="extra_javascript" content="/theme/scripts/img-gallery-handler.js">
```

Comment:
This file needs to live under `content/` because it is not part of the template.